tracer.go

```diff
@@ -175,12 +175,16 @@ func (t *tracer) startSpanWithOptions(
 
 	var samplerTags []tag
 	var ctx SpanContext
-	if !hasParent {
+	debugRequest := (hasParent && parent.traceID == 0 && parent.debugID != "")
```

Comment:
what does the the traceID == 0 check signify? does that just mean that debug spanContexts are propagated with the debugID always set to 0?